ui/src/app/utils/audit-utils.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import {
+  AuditAction,
+  AuditEventBundle,
+  AuditEventBundleHeader,
+  AuditTarget,
+  AuditTargetPropertyChange
+} from '../../generated';
+
+// Type sold separately
+// Type sold separately
```

Comment:
You probably didn't mean to do this twice and you probably also should explain what you mean by this

Reply:
that's left over. I'll collapse this whole file back into one of the others since there's only one place we need to do this now.

Reply:
Actually there was enough reuse to keep this file. Cleaned up the old comments and dead method.